src/corner_kick/common.d.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * This file defines supported file extensions
+ */
+
+declare module '*.css';
+declare module '*.png';
+declare module '*.jpg';
+declare module '*.svg';
```

Comment:
What about `.ts`, `.tsx`, `.js`, etc.? Also why is this file needed in the first place?

Reply:
Clarified into a comment
```
/**
 * This file defines additional extensions that could be present in Typescipt code and
 * are supported by our Webpack configuration. Without defining them, Typescript will
 * throw a transpile time error when we try to import a file of this type.
 */
```